app/controllers/observer_controller/create_and_edit_observation.rb

```diff
@@ -792,4 +863,15 @@ def whitelisted_observation_params
 
     params[:observation].permit(whitelisted_observation_args)
   end
+
+  def geolocation(params)
+    {
+      country:   params[:country],
+      state:     params[:state],
+      county:    params[:county],
```

Comment:
Errr... anyone care to explain to me how these aren't already aligned??

Reply:
I suggest either:
- ignore this for now, and when you're ready to merge, tell Codeclimate to Accept (I think that's the right word) it;
- Tell CodeClimate `wontfix` for **each** allegedly misaligned line
- `rubocop:disable Layout/HashAlignment` (and later `enable`) around this Hash

The problem is that we use
```yml
Layout/HashAlignment:
  EnforcedColonStyle: key
```
which demands "left align keys, **one space before ... values**" which @mo-nathan likes, 
rather than
```yml
  EnforcedColonStyle: table
```
which demands "left align keys ... and values", which you like.
See generally https://docs.rubocop.org/rubocop/1.1/cops_layout.html#layouthashalignment
Unfortunately, there doesn't seem to be a way to get RuboCop to accept multiple styles.

Reply:
Oh!  Okay, no, that's fine.  The error message said only that the *keys* need to be aligned, so it didn't occur to me that the values should *not* be aligned.  I'm fine with that.  I literally just didn't understand what it wanted me to do. :)